rfc-119-merge-mainstream-browse-topics-and-taxons.md

```diff
@@ -0,0 +1,169 @@
+
+- `title`
+- `description`
+- `analytics_identifier`
+- `content_id`
+- `document_type`
+- `first_published_at`
+- `locale`
+- `phase`
+- `public_updated_at`
+- `publishing_scheduled_at`
+- `rendering_app`
+- `scheduled_publishing_delay_seconds`
+- `schema_name`
+- `updated_at`
+- `withdrawn_notice`
+- `publishing_request_id`
+
+This is because they don't cause conflict or the mainstream or specialist topic value becomes redundant.
+
+- `publishing_app`
+
+Currently references Collections Publisher for mainstream and specialist topics, however references Content Tagger for taxons. We propose that `publishing-app` remains Content Tagger as it still has the responsibility of creating and removing taxons, even though some sub properties (i.e. links and details) may be set by Collections Publisher. In the future, we could also look at consolidating the publishing functionality of Collections Publisher and Content Tagger.
+
+- `base_path`
+
+Mainstream browse and specialist topic pages have different `base_path` to taxons. We propose that taxons keep their existing `base_path`. We could use new mainstream browse/specialist topic links within taxons to generate old base paths or find the appropriate taxon from an existing mainstream browse or specialist topic base path. Alternatively, we keep the old base paths as renamed attributes (e.g. `mainstream_browse_base_path`).
+
+#### Link attributes
+
+Taxons continue to have the following links:
+- `legacy_taxons` (potentially remove once specialist topics are removed)
+- `parent_taxons`
+- `available_translations`
+
+Add from mainstream browse:
+- `browse_parent` (parent taxons which should be used for mainstream browse)
+- `browse_children` (reverse links to `browse_parent`)
+
+Add from specialist topic:
+- `specialist_topic_parent` (parent taxons which should be used for specialist topics)
+- `specialist_topic_children` (reverse links to `specialist_topic_parent`)
+
+Dropped from mainstream browse:
+- `top_level_browse_pages` (an be derived from `base_path` and following `browse_parent`)
+- `active_top_level_browse_page` (can be derived from `base_path`)
+- `second_level_browse_pages` (an be derived from `base_path` and following `browse_parent`)
+- `related_topics` (is the related specialist topic, and would become a self reference)
+- `primary_publishing_organisation` (redundant)
+- `available_translations` (redundant)
+
+Dropped from specialist topic:
+- `taxons` (is the taxon, would become a self reference)
+- `topic_taxonomy_taxons` (redundant)
+- `primary_publishing_organisation` (redundant)
+- `available_translations` (redundant)
+
+#### Details attributes
+
+Taxons continue to have the following details:
+- `internal_name`
+- `notes_for_editors`
+- `visible_to_departmental_editors`
+
+Added from mainstream browse:
+- `browse_groups` (the curated groupings of browse children)
+
+Added from specialist topic:
+- `specialist_topic_groups` (the curated groupings of specialist_topic_children)
```

Comment:
Since we're merging these in, we have an opportunity to give them a more meaningful name - so we don't have to explain what they are.

Could we rename them to clarify that they're "overrides" of some kind?